common/templates/recordset.html

```diff
@@ -1,4 +1,17 @@
 <div ng-show="vm.initialized" class="row" style="margin-bottom: 10px;">
+    <div ng-if="vm.selectedRows.length > 0" class="row total-filters">
```

Comment:
Can you add a configuration to disable showing these? In case of faceting entity picker, we don't want to show the selected rows here and instead on the panel title.